src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -289,7 +309,7 @@ void BaseQNAcceleration::performAcceleration(
   // assume data structures associated with the LS system can be updated easily.
 
   // scale data values (and secondary data values)
-  concatenateCouplingData(cplData, _dataIDs, _values, _oldValues);
+  Acceleration::concatenateCouplingData(cplData, _dataIDs, _values, _oldValues);
```

Comment:
To make things more explicit? I think we don't do it like this most of the times, but not sure whether this is good practice.

Reply:
Yes and also to make things easier to find for me, since I initially would search for such a function in the QN classes. I can remove it again if this does not improve the readability?